jsx-a11y.js

```diff
@@ -6,10 +6,6 @@ module.exports = {
     'plugin:jsx-a11y/recommended'
   ],
   rules : {
-    'jsx-a11y/label-has-for' : [2, {
-      required : {
-        some : ['nesting', 'id']
-      }
-    }]
+    'jsx-a11y/label-has-for' : 'off' // [Deprecated on eslint-plugin-jsx-a11y v6.1.0]
```

Comment:
@masuP9 Do not use `label-has-associated-control`?

https://github.com/evcohen/eslint-plugin-jsx-a11y/releases/tag/v6.1.0

>[deprecate] The rule label-has-for is deprecated and replaced with label-has-associated-control

Reply:
@tommy-san `label-has-associated-control` rule is already include recommend on v6.1.0

https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/src/index.js#L77

Reply:
Sorry. Contained both...

https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/src/index.js#L76-L77